doc/REQUIREMENTS.MD

```diff
@@ -1,3 +1,18 @@
+- Create order API (called from `CREATEORDER_ENDPOINT_URL`) should be implemented as a GET service which accepts `encryptedCartId` query parameter with Blowfish encrypted cart id string, for instance:
+
+  ```
+  http://localhost:9000/API/order/createOrder?encryptedCartId=eDIeqi7seRTjdYRqm%2BOxy5Qcbc38dXaEgftHjQcr6M7HO125Tg%2BnUg%3D%3D
+  ```
```

Comment:
i'm just curious why in URL we should prefer base64 encoding over native URL encoding? Also: on service side base64 will require additional decode step, when with native URLencoding i just use value come from route controller:
```
public CompletionStage<Result> createOrder(@Nonnull String encryptedCartId) {
```
instead of 
```
public CompletionStage<Result> createOrder(@Nonnull String encryptedBase64CartId) {
    encryptedCartId = new String(Base64.getUrlDecoder().decode(encryptedBase64CartId.getBytes()));
```

IMHO in this certain step base64 encoding is superfluous (it might have sense if we send value in HTTP body)